src/Sulu/Bundle/AdminBundle/Admin/AdminPool.php

```diff
@@ -25,10 +25,8 @@ class AdminPool
 
     /**
      * Adds a new admin.
-     *
-     * @param $admin
      */
-    public function addAdmin($admin)
+    public function addAdmin(Admin $admin)
```

Comment:
i think we cannot add typehints here because of backwards compatibility. if somebody extended this class and overwrote this method, his application will break because the method signature does not match anymore 😕

Reply:
Aggree here we can from BC promise not add typehints here. This should be added to the phpdoc above:

```
     * @param Admin $admin
```